src/app/health-page/health-panel/health-component/health-component.component.ts

```diff
@@ -4,6 +4,9 @@ import { HealthComponent } from '../../models/health-component.model';
 import { TranslateService } from '@ngx-translate/core';
 import { AlertType } from '../../../shared/alert/aletr-type';
 
+/**
+ * Show a health component object
+ */
```

Comment:
That doesn't clarify very much. I was thinking something along the lines of:

```suggestion
  /**
   * A component to render a "health component" object.
   *
   * Note that the word "component" in "health component" doesn't refer to Angular's use of the term
   * but rather to the components used in the response of the health endpoint of Spring's Actuator 
   * API.
   */
```

I'd do something similar for HealthInfoComponentComponent, as well as the two models themselves.

Reply:
done